clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -299,6 +299,10 @@ Miscellaneous
   ``--format`` option is specified. Now :program:`clang-apply-replacements`
   applies formatting only with the option.
 
+- Fixed the :doc:`linuxkernel-must-check-errs
+  <clang-tidy/checks/linuxkernel/must-check-errs>` documentation to consistently
+  use the check's proper name.
```

Comment:
No release notes needed for documentation changes.

Reply:
Are you sure it's not worth it? The documentation has been misleading for almost 5 years at this point.

Reply:
And none one cared. If you really like to keep it then fine, I'm not pushing but usually in change notes we got only changes in checks behaviour.